client/components/Windows/Settings.vue

```diff
@@ -351,16 +351,45 @@ This may break orientation if your browser does not support that."
 
 			<div v-if="!$store.state.serverConfiguration.public && $store.state.settings.advanced">
 				<label class="opt">
-					<label for="highlights" class="sr-only">
-						Custom highlights (comma-separated keywords)
+					<label for="highlights" class="opt">
+						Custom highlights
+						<span
+							class="tooltipped tooltipped-n tooltipped-no-delay"
+							aria-label="If a message contains any of these comma-separated expressions, it will trigger a highlight."
+						>
+							<button class="extra-help" />
+						</span>
 					</label>
 					<input
 						id="highlights"
 						:value="$store.state.settings.highlights"
 						type="text"
 						name="highlights"
 						class="input"
-						placeholder="Custom highlights (comma-separated keywords)"
+						placeholder="word, some more words, anotherword"
```

Comment:
Shall we keep comma-separated here to make it clearer?

Like `Comma-separated, for example: word, some more words, anotherword` or something.

Reply:
Maybe. We can use "e.g." so the text fits the field size:
![image](https://user-images.githubusercontent.com/2070503/88397372-1baf9f80-cdc4-11ea-863d-a11abce310db.png)